portfolio/src/main/webapp/script.js

```diff
@@ -66,6 +66,38 @@ function showProjects() {
     }
 }
 
+var logStatus;
+var temp;
+
+
+/** Initiate the home page */
+async function loadHomePage() {
+    const inputForm = document.getElementById("input-form");
+    const log = document.getElementById("logging");
+    const link = document.getElementById("log-link");
+
+    await getComments();
+    logStatus = (await getLogStatus() !== 'false'); 
```

Comment:
declare logStatus here.

```suggestion
    const logStatus = (await getLogStatus() !== 'false'); 
```

Reply:
wouldnt const mean that logStatus value can not change?

Reply:
Oh is it because the variable only updates on refresh of the page so it doesnt change anyways?